recipes/eccodes-python/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set name = "eccodes-python" %}
+{% set pypi_name = "eccodes-python" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: 587dd828ee5e13b6a7d35a362d6fd903d914dd0422f4010af95e8f45738e9143
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  commands:
+    - test -f $(python -c "import site; print(site.getsitepackages()[0])")/eccodes/__init__.py  # [not win]
+
+about:
+  home: https://github.com/ecmwf/eccodes-python
+  license: BSD 3
```

The file and family should also be fixed
```suggestion
  license: Apache-2.0
```

---

Done